File: El Mamoun/Lv 1/Problem 46.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

/*
Problem 46: Write a program to print all letters from A to Z

*/

namespace Problem_46
{
    internal class Program
    {

        static void PrintAllLetterFromAtoZ()
        {

            for(int i= 0; i < 26; i++)
            {

                Console.WriteLine(Convert.ToChar(i + (int)'B'));
            }
        }


        static void Main(string[] args)
        {

            PrintAllLetterFromAtoZ();

            Console.ReadKey();
        }
    }
}

File: El Mamoun/Lv 1/Problem 46.cpp
#include<iostream>
using namespace std;

/*
Problem 46: Write a program to print all letters from A to Z
*/

void PrintAllLetterFromAToZ()
{

	for (short i = 65; i <= 90; i++)
	{
		cout << char(i) << endl;
	}
}


int main()
{

	PrintAllLetterFromAToZ();

	return 0;
}